src/templates/options.js

```diff
@@ -19,3 +19,9 @@ for (const [key, value] of Object.entries(options)) {
     }
 }
 %>
+
+<% if (options.langDir) { %>
+export const ASYNC_LOCALES = {
+  <%= options.locales.map(l => `'${l.file}': () => import('../${relativeToBuild(options.langDir, l.file)}' /* webpackChunkName: "lang-${l.file}" */)`).join(',\n  ') %>
```

Comment:
If multiple locales reuse the same `file` then now the object will contain repeated and redundant properties.